src/training/mod.rs

```diff
@@ -214,14 +284,13 @@ extern "C" {
 }
 
 #[skyline::hook(replace = set_dead_rumble)]
-pub unsafe fn handle_set_dead_rumble(
-    lua_state: u64) -> u64 {
-
-    if crate::common::is_training_mode() {
-        return 0;
+pub unsafe fn handle_set_dead_rumble(lua_state: u64) -> u64 {
```

Comment:
This one is wrong, we don't want the original to be called if we are in training mode at all, not just ignore the return value.